conf/pika.conf

```diff
@@ -1,141 +1,280 @@
-# Pika port
+###########################
+# Pika configuration file #
+###########################
+
+# Pika port, the default value is 9221
```

Comment:
maybe we could mention about the +100/+200 magic offsets here if it is still valid.